pkg/providers/vsphere/reconciler/reconciler.go

```diff
@@ -236,26 +224,6 @@ func (r *Reconciler) ReconcileWorkers(ctx context.Context, log logr.Logger, spec
 	return clusters.ReconcileWorkersForEKSA(ctx, log, r.client, spec.Cluster, clusters.ToWorkers(w))
 }
 
-// InstallStorageClass install default storage class in workload cluster.
-func (r *Reconciler) InstallStorageClass(ctx context.Context, log logr.Logger, clusterSpec *c.Spec) (controller.Result, error) {
```

Comment:
Are we removing the `GenerateStorageClass()` or `GenerateDefaultStorageClass` methods from all the providers as a result of this change? If not, should we have an issue to track that?

Reply:
I saw the storageClass method only in the Nutanix provider and [it is currently a no-op](https://github.com/aws/eks-anywhere/blob/main/pkg/providers/nutanix/provider.go#L499-L502) so I think we should just remove it now and add it later if needed.[](url)